woocommerce/woocommerce-ios
# Blaze: Create campaign from the payment screen

Closes: #11702 


## Description

This PR integrates campaign creation on the payment screen when a selected payment method is available.

The loading state has been handled on the UI, and the callback closure is triggered upon success. The success bottom sheet and error state should be handled in separate PRs.

## Testing instructions


- Log in to a store that's eligible for Blaze.
- If the campaign creation entry point is available on the Dashboard screen, select it and select a product if needed. Otherwise, navigate to the Products tab, select a published product and tap Promote with Blaze.
- On the campaign creation form, tap Confirm Details.
- On the payment screen, tap Submit Campaign.
- Confirm that a loading screen is displayed briefly.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/545ecc50-7b90-489c-b447-2f267c54ef15



---
